.github/workflows/buildandpublish.yml

```
@@ -0,0 +1,32 @@
+name: Build and publish Ansible
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  build-and-publish:
+    name: Build and publish Ansible to PyPI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
```

v3 relies on the deprecated NodeJS 12, don't use it.
```suggestion
    - name: Set up Python 3.11
      uses: actions/setup-python@v4
      with:
        python-version: 3.11
```